domain/src/main/assets/umPkwp0L1M0-.json

```diff
@@ -271,19 +265,18 @@
                 "labelled_as_correct": false
               },
               "training_data": [],
-              "rule_specs": [
-                {
-                  "rule_type": "IsExactlyEqualTo",
-                  "inputs": {
+              "rule_types_to_inputs": {
+                "IsExactlyEqualTo": [
+                  {
                     "f": {
-                      "denominator": 7,
+                      "denominator": 3,
```

Comment:
Why has this changed?

Reply:
I have checked it on develop and the denominator should be 7. Thanks for catching this.